clinica/iotools/bids_readme.py

```diff
@@ -0,0 +1,72 @@
+from enum import Enum
+from typing import IO
+
+from attrs import define, fields
+from cattr.gen import make_dict_unstructure_fn, override
+from cattr.preconf.json import make_converter
+
+BIDS_VERSION = "1.7.0"
+
+
+@define
+class BIDSReadme:
+    """Model representing a BIDS ReadMe.
+
+    See
+    """
+
+    class DatasetType(str, Enum):
+        raw = "raw"
+        derivative = "derivative"
+
+    name: str
```

Comment:
The class attributes should reflect what is needed for the BIDSReadme

Reply:
Done !

Reply:
Currently it still doesn't really. I still see a `bids_name` and `bids_version` which make sense for ther `dataset_description.json` but are not attributes of the readme.